packages/connection/legacy/class.jetpack-signature.php

```diff
@@ -1,30 +1,27 @@
 <?php
 
-// These constants can be set in wp-config.php to ensure sites behind proxies will still work.
```

Comment:
Can we add these comments back to the new `define()`s in jetpack.php? I want it to remain clear that these constants are not a good way to solve the problem :)